### ISSUE TITLE
Setup AWS Profile

### ISSUE THREAD

**Issue by acevedomiguel** [indexsoftware/vscode-rds-data-api#5]
I'm not using the default aws profile for the client:

```
Command failed: aws rds describe-db-clusters An error occurred (InvalidClientTokenId) when calling the DescribeDBClusters operation: The security token included in the request is invalid.
```

Thanks

**Comment by joostdelange** [comment]
Hi @acevedomiguel, thanks a lot for the issue. I'll come up with a fix later today or tomorrow.

**Comment by joostdelange** [comment]
@acevedomiguel

A new command _Select AWS profile_ is now available in v0.0.8, which allows you to set the profile you want to use. Please let me know if there's any issues!

**Comment by acevedomiguel** [comment]
Thanks a lot!
